src/modules/item/starred/starred.controller.ts

```diff
@@ -0,0 +1,90 @@
+import { FastifyReply, FastifyRequest } from 'fastify';
+import { ReadInput, AddInput, DeleteInput } from './starred.schema';
+import StarredService from './starred.service';
+import AccessService from '../sharing/access.service';
+
+export default class StarredController {
```

Comment:
Add extra tests to make sure you can read/remove starrings, which are on items you don't have access to